### ISSUE TITLE
Moving filtered tabs issue

### ISSUE THREAD

**Issue by ZbynekT** [Rapptz/power-tabs#35]
If user wants to move/drag filtered tabs, the tabs which are hidden and in-between moved tabs are moved as well.
Repro:
1.Open 2 tabs from domain X
2.Open 2 tabs from domain Y
3.Open 2 tabs from domain X
*the tab order should be XXYYXX
4. Create the second empty Group
5.Filter domain X (search)
6.Select all domain X tabs (click the first and with Shift, select the last tab)
7.Drag tabs of domain X to empty Group
8.Clean filter
